src/app/frontend/deploy/deploy_controller.js

```diff
@@ -69,6 +69,12 @@ export default class DeployController {
      * @export {{deploy:function()}}
      */
     this.detail = {deploy: function() {}};
+
+    /**
+     * child directive selection model
```

Comment:
Add a note that the list of possible values is in the template. And this value represents default selection.